lib/miq_automation_engine/engine/miq_ae_service/miq_ae_service_rbac.rb

```diff
@@ -0,0 +1,34 @@
+module MiqAeMethodService
+  module MiqAeServiceRbac
+    extend ActiveSupport::Concern
+
+    module ClassMethods
+      def find_ar_object_by_id(id)
+        Rbac.search(:class => model, :targets => [id],
+                    :results_format => :objects).try(:first).try(:first)
+      end
+
+      def all
+        wrap_results(Rbac.search(:class => model, :results_format => :objects).try(:first))
+      end
+
+      def count
+        Rbac.search(:class => model).try(:first).count
+      end
+
+      def first
+        all.first
+      end
+
+      def filter_objects(objs)
+        return objs if objs.nil?
```

Comment:
```ruby
def filter_objects(objs)
  objs.nil? ? nil : Rbac.filtered(Array.wrap(objs)
end
```

Note: please keep the nil check there even though it will work without
trying to get filtered to treat a nil input as a "search for stuff"